test/lit.cfg

```diff
@@ -905,6 +909,16 @@ elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'wi
       config.target_sdk_name = "linux"
     config.target_runtime = "native"
     config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
+
+    config.libdispatch_build_dir = make_path(config.swift_obj_root, os.pardir,
+        'libdispatch%s' % config.variant_suffix, 'src')
+    if os.path.exists(config.libdispatch_build_dir):
```

Comment:
I think that you should make this more resilient and ensure that dispatch, swiftDispatch, and the swift module are available before declaring the feature as being available.

Reply:
Good point. I will try to make this more resilient.